src/Providers/DocumentCreation/Generic/TextfileDocumentCreator.cs

```diff
@@ -24,13 +24,13 @@ public byte[] CreateDocument(List<string> fileContent)
             return stream.ToArray();
         }
 
-        public byte[] CreateHtmlDocument(List<string> fileContent)
+        public byte[] CreateHtmlDocument(List<string> fileContent, string formName)
         {
             using (var stream = new MemoryStream())
             {
                 var objStreamWriter = new StreamWriter(stream);
                 objStreamWriter.WriteLine("<div>");
-                objStreamWriter.WriteLine("<h2>Form answers</h2>");
+                objStreamWriter.WriteLine("<h2>Submitted answers for {0}</h2>", formName);
```

Comment:
Is it possible to do the interpolation like this instead?

$"\<h2>Submitted answers for {formName}\</h2>"